src/utils.ts

```diff
@@ -1,22 +1,26 @@
-function hueRotationFn(rotation) {
-  return (hue) => (360 + hue + rotation) % 360;
+import {Config} from './types';
+
+export function hueRotationFn(rotation: number) {
+  return (hue: number) => (360 + hue + rotation) % 360;
 }
 
-function saturationAdjustmentFn(adjustment) {
-  return (saturation) => Math.min(Math.max(saturation + adjustment, 0), 100);
+export function saturationAdjustmentFn(adjustment: number) {
+  return (saturation: number) =>
+    Math.min(Math.max(saturation + adjustment, 0), 100);
 }
 
-function tokensToJson(data) {
+// TODO: type data
```

Comment:
We'll need to address this one, but I still need to wrap my head around the shape of `data`